util/feedback.py

```diff
@@ -0,0 +1,25 @@
+import uuid
+from starlette.exceptions import HTTPException
+from config.db import get_database
+import logging
+DB_NAME = "underline"
+
+# Given an event id and feedback id, attempt to delete the feedback from the event's comment_ids array
+async def delete_feedback(event_id, feedback_id, db):
+    
+    column = db[DB_NAME]["events"]
+
+    found_event = column.find_one({"_id": event_id})
+    
+    if not found_event:
+        raise HTTPException(status_code=404,
+                            detail="Event ID is invalid")
+
+
+    if feedback_id in found_event["comment_ids"]:
+        found_event["comment_ids"].remove(feedback_id)
+    else:
+        raise HTTPException(status_code=404,
+                            detail="Feedback ID not found in the provided event")
+                            
+    column.update_one({"_id": event_id}, {"$set": found_event})
```

Comment:
This only deletes the feedback from the event document, which is fine as far as the user is concerned, but leaves an orphan feedback document with not pointer behind.

The last operation in the method should be to delete the feedback from the feedback collection

Reply:
I didn't write any feedback code, I'll have to look into how I got this on my branch😂

Reply:
Wait what hahahahah no wonder it looked so out of place hahaha. No worries then just ignore.